### ISSUE TITLE
Fine-tuning  ASR (stt_en_conformer) on another language.

### ISSUE THREAD

**Issue by gabitza-tech** [NVIDIA/NeMo#6871]
Hello,

I would like to use the pre-trained stt_en_conformer_ctc or stt_en_conformer_transducer models for transfer learning on another language ( I only have around 1000 hours of audio, and I would like to jump-start the training/try to see if it may bring better results). I am new to ASR, so I am not very familiar if this is possible. From my understanding, I could use the pre-trained weights from the encoder, and only change the tokens of the decoder ( but the tokenizer should have the same size as the pretrained model, even if the tokens are different). 

I would greatly appreciate some clarifications/corrections to what I mentioned above and if someone could guide me towards a tutorial/example that tackles something similar?

Thank you in advance!

**Comment by titu1994** [comment]
Please refer to the ASR tutorials section - it describes steps to fine tune models. 

If you are training on another language, you need to modify the tokenizer as well. Even for the same language, you may need a new Tokenizer otherwise you'll need to normalize the text to lower case and remove all special characters other than space and apostrophe.

**Comment by titu1994** [comment]
Woops. Here is link to the tutorial - https://colab.research.google.com/github/NVIDIA/NeMo/blob/stable/tutorials/asr/ASR_CTC_Language_Finetuning.ipynb

The same steps are for transducer as well.

**Comment by gabitza-tech** [comment]
Thank you very much @titu1994 !
